packages/agreement-process/src/model/domain/validators.ts

```diff
@@ -203,17 +201,10 @@ export const verifyCreationConflictingAgreements = async (
   agreement: ApiAgreementPayload,
   agreementQuery: AgreementQuery
 ): Promise<void> => {
-  const conflictingStates: AgreementState[] = [
```

Comment:
moved this to `models` as `agreementCreationConflictingStates` to reuse it in the unit test

Reply:
this definition is not part of the model itself but it is strictly related to the business logic, it should probably be in the service that uses it.
What do you think?

Reply:
Agree! We are moving all these changes to a new PR that implements these tests as integration tests, not unit tests - and we are also going to address this: #176